v21.1/known-limitations.md

```diff
@@ -6,6 +6,159 @@ toc: true
 
 This page describes newly identified limitations in the CockroachDB {{page.release_info.version}} release as well as unresolved limitations identified in earlier releases.
 
+## New limitations
+
+### CockroachDB does not properly optimize some left and inverted joins with inverted indexes
+
+[Left joins](joins.html#left-outer-joins) and [inverted joins](joins.html#inverted-joins) involving [`JSONB`](jsonb.html), [`ARRAY`](array.html), or [spatial-typed](spatial-data.html) columns with a multi-column or [partitioned](partition-by.html) [inverted index](inverted-indexes.html) will not take advantage of the index if the prefix columns of the index are unconstrained, or if they are constrained to multiple, constant values.
+
+To work around this limitation, make sure that the prefix columns of the index are either constrained to single constant values, or are the same as the input columns.
```

Comment:
nit: "the same as input columns" is misleading -- I think this should be referring to equality conditions. If the prefix columns are not held constant, they should be part of equality conditions (e.g., col1 = col2), where col1 is a prefix column and col2 is an input column.